packages/runtime/src/components/createElement.ts

```diff
@@ -158,35 +161,52 @@ export function createElement({
     elem.addEventListener(event.trigger, handler)
   })
 
-  // for script and style tags we just render the first text child.
+  // for script, style & SVG<text> tags we only render text child.
```

Comment:
It probably doesn't matter, but you could avoid (potentially) calling the same function 3 x by extracting it to a const. Also, I suggest updating the comment
```suggestion
  // for script, style & SVG<text> tags we only render text children
  const nodeTag = node.tag.toLocaleLowerCase()
```

Reply:
good suggestion. I wanted to do a string array, but `text` is dependent on it being an SVG, so it is more logic than a static lookup. Your suggestion helps on readablity 👍